### ISSUE TITLE
Thick grey bar appears above tabs when changing focus in Full Screen mode (macOS)

### ISSUE THREAD

**Issue by autonomic** [brave/brave-browser#7623]
## Description 
When running Brave in Full Screen mode on macOS and then changing focus to another application (⌘ + tab), the tab bar (at the top of the window) gets pushed down by a thick grey bar (screenshots attached).

## Steps to Reproduce

   1. Open browser
   2. Click on the Full Screen button in the top left corner (displays on 1 of 2 monitors)
   3. Hit ⌘ + tab to change focus to another app
   4. You'll now see that the whole browser shifts downwards and there is a grey bar (section above it)

## Actual result:

![image](https://user-images.githubusercontent.com/13229225/71861707-20249d00-30ac-11ea-8bf6-f2f13a51f191.png)

## Expected result:
![image](https://user-images.githubusercontent.com/13229225/71861784-67129280-30ac-11ea-8da6-0f9c5cf24ea2.png)

## Reproduces how often: 

I can easily reproduce every time

## Brave version (brave://version info)

Brave | 1.2.41 Chromium: 79.0.3945.88 (Official Build) (64-bit)
Revision | c2a58a36b9411c80829b4b154bfcab97e581f1f3-refs/branch-heads/3945@{#954}
OS | macOS Version 10.14.6 (Build 18G2022)

## Version/Channel Information:

- Can you reproduce this issue with the current release?
Yes I'm on the current release

- Can you reproduce this issue with the beta channel?
- Can you reproduce this issue with the dev channel?
- Can you reproduce this issue with the nightly channel?

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields?
It's the entire window, not a web page

- Does the issue resolve itself when disabling Brave Rewards?
It's the entire window, not a web page

- Is the issue reproducible on the latest version of Chrome? 
Doesn't happen with Chrome (see screenshot below)
![image](https://user-images.githubusercontent.com/13229225/71861950-fa4bc800-30ac-11ea-9353-0ace0b886cc0.png)

**Comment by rebron** [comment]
@autonomic We're not able to reproduce this, can you try on the latest version, 1.3.x?

**Comment by johnaoss** [comment]
I think this is/was a bug in Chromium. I've seen this pretty frequently at work, but it may have been fixed in the latest version.

**Comment by autonomic** [comment]
@rebron @johnaoss This only happens when I'm at the office connected to my two monitors with my Macbook Pro in clamshell mode. Is there something I can do to provide additional debug information?

**Comment by johnaoss** [comment]
@autonomic If you can find steps to reproduce this that would be the best bet.

Aside from that, there exists a [guide on debugging Brave UI](https://github.com/brave/brave-browser/wiki/Debugging-Brave-UI) however as I'm not a Brave dev I'm not sure what sort of information would be helpful to them.

**Comment by autonomic** [comment]
@rebron So I'm running the latest stable release of Brave (1.4.96 Chromium: 80.0.3987.132 (Official Build) (64-bit)). I have my Macbook Pro Retina display as one screen and an external display as my second screen. These are the steps I use to reproduce.

1. Open up Brave in the native fullscreen mode in macOS and have it running on the external display
2. Open a second application (in my case MacVim) and have it running as a regular window on the Macbook Retina display
3. Navigate to a webpage in Brave and then use your mouse cursor to click somewhere on the page
4. Use the Command + Tab keys to cycle to the application you opened in step # 2.
5. When the application on the Macbook display takes focus, I see the Brave browser shift down and the top section that normally is right above the tabs, grown in size

It's important to note this wont work if you are just using your mouse to click between apps on either screen. You have to use your mouse to click somewhere in the Brave browser window or current web page and then use the Command + Tab key to change to the app on the Macbook display.

Cheers,
-Chris

**Comment by bsclifton** [comment]
@autonomic sorry that we haven't been able to make any progress here ☹️ I am curious if you're still experiencing this with the latest version of Brave. Would you be able to update and let us know? Thanks!

**Comment by autonomic** [comment]
@bsclifton Yes, still getting this behavior and I'm surprised more people aren't complaining as I mustn't be the only one. I'm running the latest version of Brave (Version 1.23.71 Chromium: 90.0.4430.72 (Official Build) (x86_64)).

I've attached a video to show you. I have my MacBook Pro retina display open as my first display and an external display connected to my Thunderbolt 3 dock as my second display. Since my initial post I have upgraded my Mac to a 2020 MacBook Pro so it's not something specific to the hardware.

In the video the browser is in focus on the external display. I then use Launchbar to open MacVim on my MacBook retina display. As soon as MacVim opens on the laptops retina display you can see the browser pushes down and there is a thick grey bar at the top.


https://user-images.githubusercontent.com/13229225/115730932-913ca200-a33b-11eb-81f8-accb094e7da6.mov

**Comment by adjoseph16** [comment]
Hi there, I know it's been an incredibly long time but I am also experiencing this issue. Happy to help however I can. I think it's related to this setting being disabled, as I only just disabled it a few days ago and that's when the issue began.

<img width="657" alt="image" src="https://user-images.githubusercontent.com/78319881/190279888-7aea964e-84e3-4181-8478-b7b64381c869.png">


Brave Version 1.43.93 Chromium: 105.0.5195.127 (Official Build) (x86_64)
MacOS Monterey 12.4 (21F79)

**Comment by MathKlone** [comment]
Solution I found is to swipe up with three fingers (mission Control gesture) and swipe back down. That gets rid of the extra space.

**Comment by amochkin** [comment]
Experiencing the same issue. It is spontaneous. 
MacBook Pro M2 (please note it has camera block inside screen rectangle).
As a side note, Firefox does not have this issue.

**Comment by Onn-dre** [comment]
Im having the same issue, 2019 macbook with a second monitor. Switching workspaces back and forth gets rid of it, but Im surprised this has been going on so long without a permanent fix.

**Comment by Marcoj776** [comment]
Yes, what I've been doing is toggling fullscreen on both screens until it
goes away
